### ISSUE TITLE
Incremental build rebuilds all the packages

### ISSUE THREAD

**Issue by travisn** [rook/rook#349]
Rook rebuilds all the packages with incremental builds even if nothing changed. The incremental build needs to be very fast for dev iteration. This happens for rookd, but is now more pronounced in my branch with the new rook-operator.
```
tnielsen@ubuntu:~/src/go/src/github.com/rook/rook$ time build/run make -j4
[  0%] Built target common_utf8
[  0%] Built target common_mountcephfs_objs
[  0%] Built target compressor_objs
[  0%] Built target mon_common_objs
[  0%] Built target common_texttable_obj
[  0%] Built target common_buffer_obj
[  0%] Built target erasure_code
[  0%] Built target global_common_objs
[  2%] Built target json_spirit
[  6%] Built target liblua
[  6%] Built target cls_journal_client
[  6%] Built target cls_rgw_client
[  6%] Built target cls_version_client
[  6%] Built target cephd_base
[  6%] Built target cls_statelog_client
[  9%] Built target cls_log_client
[  9%] Built target cls_lock_client
[  9%] Built target cls_refcount_client
[  9%] Built target cls_timeindex_client
[  9%] Built target cls_replica_log_client
[ 11%] Built target osdc_rbd_objs
[ 13%] Built target cls_user_client
[ 13%] Built target librados_api_obj
[ 13%] Built target librados_objs
[ 13%] Built target kv_objs
[ 13%] Built target heap_profiler_objs
[ 13%] Built target kv
[ 13%] Built target common_util_obj
[ 13%] Built target cls_references_objs
[ 13%] Built target erasure_code_objs
[ 13%] Built target osd_mon_objs
[ 20%] Built target cephd_ec_isa
[ 22%] Built target gf-complete_objs
[ 22%] Built target jerasure_objs
[ 22%] keep civetweb.h up-to-date
[ 22%] Built target civetweb_h
[ 22%] Built target cephd_ec_lrc
[ 22%] Built target civetweb_common_objs
[ 22%] Built target cephd_ec_shec
[ 22%] Built target cephd_compressor_base
[ 25%] Built target cephd_compressor_zlib
[ 27%] Built target cephd_compressor_snappy
[ 27%] Built target cephd_rbd_base
[ 27%] Built target cephd_rgw_base
[ 27%] Built target rados_snap_set_diff_obj
[ 27%] Built target rbd_types
[ 27%] Built target cephd_cls_kvs
[ 27%] Built target journal
[ 27%] Built target cephd_rados
[ 27%] Built target osdc
[ 34%] Built target cephd_cls_base
[ 34%] Built target cephd_ec_base
[ 34%] Built target cephd_ec_jerasure
[ 34%] Built target cephd_compressor
[ 36%] Built target mon
[ 36%] Built target cephd_cls
[ 36%] Built target cephd_ec
[ 38%] Built target osd
[ 50%] Built target rbd_internal
[ 52%] Built target cephd_rbd
[ 72%] Built target common
[ 72%] Built target librados
[ 72%] Built target ceph_snappy
[ 75%] Built target ceph_zlib
[ 75%] Built target global
[ 75%] Built target compressor_plugins
[ 79%] Built target os
[ 86%] Built target mds
[ 97%] Built target rgw_a
[100%] Built target radosgw_a
[100%] Built target cephd_rgw
[100%] Built target cephd
github.com/rook/rook/vendor/github.com/coreos/go-systemd/journal
github.com/rook/rook/pkg/model
github.com/rook/rook/vendor/github.com/spf13/pflag
github.com/rook/rook/pkg/util/display
github.com/rook/rook/vendor/github.com/google/uuid
github.com/rook/rook/pkg/version
github.com/rook/rook/cmd/rook/client
github.com/rook/rook/pkg/rook/client
github.com/rook/rook/vendor/github.com/coreos/pkg/capnslog
github.com/rook/rook/pkg/util/exec
github.com/rook/rook/pkg/util/kmod
github.com/rook/rook/pkg/util/sys
github.com/rook/rook/vendor/github.com/spf13/cobra
github.com/rook/rook/pkg/util/flags
github.com/rook/rook/cmd/rook/version
github.com/rook/rook/cmd/rook/rook
github.com/rook/rook/cmd/rook/block
github.com/rook/rook/cmd/rook/filesystem
github.com/rook/rook/cmd/rook/node
github.com/rook/rook/cmd/rook/object
github.com/rook/rook/cmd/rook/pool
github.com/rook/rook/cmd/rook/status
github.com/rook/rook/cmd/rook
github.com/rook/rook
github.com/rook/rook/vendor/github.com/coreos/go-systemd/journal
github.com/rook/rook/pkg/model
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/davecgh/go-spew/spew
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/golang/glog
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/ugorji/go/codec
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/golang.org/x/net/context
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/types
github.com/rook/rook/vendor/github.com/coreos/pkg/capnslog
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/mailru/easyjson/jlexer
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/api/meta/metatypes
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/mailru/easyjson/buffer
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/PuerkitoBio/urlesc
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/mailru/easyjson/jwriter
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/golang.org/x/net/idna
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/golang.org/x/text/internal/tag
github.com/rook/rook/pkg/cephmgr/client
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/golang.org/x/text/transform
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/golang.org/x/text/unicode/bidi
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/golang.org/x/text/language
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/go-openapi/swag
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/golang.org/x/text/unicode/norm
github.com/rook/rook/pkg/cephmgr/cephd
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/golang.org/x/text/runes
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/go-openapi/jsonpointer
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/golang.org/x/text/secure/bidirule
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/golang.org/x/text/width
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/gogo/protobuf/proto
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/gogo/protobuf/sortkeys
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/google/gofuzz
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/third_party/forked/golang/reflect
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/golang.org/x/text/cases
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/selection
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/conversion
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/util/sets
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/golang.org/x/text/secure/precis
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/util/validation
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/conversion/queryparams
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/labels
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/util/errors
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/PuerkitoBio/purell
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/util/json
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/spf13/pflag
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/go-openapi/jsonreference
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/gopkg.in/inf.v0
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/go-openapi/spec
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/auth/user
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/fields
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/gopkg.in/yaml.v2
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/util/framer
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/util/rand
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/pborman/uuid
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/util/validation/field
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/util
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/util/uuid
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/docker/distribution/digest
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/genericapiserver/openapi/common
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/api/unversioned
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/api/resource
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/util/intstr
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/docker/distribution/reference
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/util/parsers
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/golang.org/x/net/http2/hpack
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/golang.org/x/net/lex/httplex
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/util/runtime
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/util/wait
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/golang.org/x/net/http2
github.com/rook/rook/vendor/github.com/coreos/etcd/pkg/pathutil
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/ghodss/yaml
github.com/rook/rook/vendor/github.com/coreos/etcd/pkg/types
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/util/yaml
github.com/rook/rook/vendor/github.com/ugorji/go/codec
github.com/rook/rook/vendor/golang.org/x/net/context
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/runtime
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/util/labels
github.com/rook/rook/vendor/github.com/go-ini/ini
github.com/rook/rook/vendor/github.com/coreos/etcd/error
github.com/rook/rook/vendor/github.com/jonboulle/clockwork
github.com/rook/rook/vendor/github.com/beorn7/perks/quantile
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/api/meta
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/runtime/serializer/recognizer
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/runtime/serializer/versioning
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/runtime/serializer/protobuf
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/api/errors
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/runtime/serializer/streaming
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/util/net
github.com/rook/rook/vendor/github.com/golang/protobuf/proto
github.com/rook/rook/vendor/bitbucket.org/ww/goautoneg
github.com/rook/rook/vendor/github.com/prometheus/common/model
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/watch
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/watch/versioned
github.com/rook/rook/vendor/github.com/prometheus/procfs
github.com/rook/rook/vendor/github.com/google/uuid
github.com/rook/rook/pkg/util/exec
github.com/rook/rook/vendor/github.com/grpc-ecosystem/grpc-gateway/utilities
github.com/rook/rook/pkg/util/sys
github.com/rook/rook/vendor/golang.org/x/net/http2/hpack
github.com/rook/rook/vendor/golang.org/x/net/internal/timeseries
github.com/rook/rook/vendor/github.com/prometheus/client_model/go
github.com/rook/rook/vendor/github.com/matttproud/golang_protobuf_extensions/pbutil
github.com/rook/rook/vendor/github.com/coreos/etcd/auth/authpb
github.com/rook/rook/vendor/github.com/coreos/etcd/mvcc/mvccpb
github.com/rook/rook/vendor/github.com/prometheus/common/expfmt
github.com/rook/rook/vendor/github.com/golang/protobuf/jsonpb
github.com/rook/rook/vendor/github.com/grpc-ecosystem/grpc-gateway/runtime/internal
github.com/rook/rook/vendor/golang.org/x/net/http2
github.com/rook/rook/vendor/golang.org/x/net/trace
github.com/rook/rook/vendor/google.golang.org/grpc/codes
github.com/rook/rook/vendor/google.golang.org/grpc/credentials
github.com/rook/rook/vendor/github.com/prometheus/client_golang/prometheus
github.com/rook/rook/vendor/google.golang.org/grpc/grpclog
github.com/rook/rook/vendor/google.golang.org/grpc/internal
github.com/rook/rook/vendor/google.golang.org/grpc/metadata
github.com/rook/rook/vendor/google.golang.org/grpc/naming
github.com/rook/rook/vendor/google.golang.org/grpc/peer
github.com/rook/rook/vendor/github.com/boltdb/bolt
github.com/rook/rook/vendor/golang.org/x/crypto/blowfish
github.com/rook/rook/vendor/golang.org/x/crypto/bcrypt
github.com/rook/rook/vendor/github.com/coreos/etcd/lease/leasepb
github.com/rook/rook/vendor/github.com/coreos/etcd/pkg/adt
github.com/rook/rook/vendor/github.com/coreos/etcd/pkg/schedule
github.com/rook/rook/vendor/github.com/google/btree
github.com/rook/rook/vendor/github.com/coreos/etcd/store
github.com/rook/rook/vendor/google.golang.org/grpc/transport
github.com/rook/rook/vendor/github.com/coreos/etcd/mvcc/backend
github.com/rook/rook/vendor/github.com/coreos/etcd/pkg/fileutil
github.com/rook/rook/vendor/github.com/coreos/etcd/pkg/tlsutil
github.com/rook/rook/vendor/github.com/coreos/etcd/pkg/transport
github.com/rook/rook/vendor/github.com/coreos/etcd/lease
github.com/rook/rook/vendor/github.com/coreos/etcd/pkg/netutil
github.com/rook/rook/vendor/github.com/coreos/etcd/mvcc
github.com/rook/rook/vendor/github.com/coreos/etcd/raft/raftpb
github.com/rook/rook/vendor/github.com/coreos/etcd/version
github.com/rook/rook/vendor/github.com/coreos/go-semver/semver
github.com/rook/rook/vendor/google.golang.org/grpc
github.com/rook/rook/vendor/github.com/coreos/etcd/etcdserver/api/v2http/httptypes
github.com/rook/rook/vendor/github.com/coreos/etcd/raft
github.com/rook/rook/vendor/github.com/coreos/etcd/pkg/contention
github.com/rook/rook/vendor/github.com/coreos/etcd/pkg/httputil
github.com/rook/rook/vendor/github.com/coreos/etcd/pkg/idutil
github.com/rook/rook/vendor/github.com/coreos/etcd/pkg/pbutil
github.com/rook/rook/vendor/github.com/coreos/etcd/pkg/runtime
github.com/rook/rook/vendor/github.com/coreos/etcd/pkg/wait
github.com/rook/rook/vendor/github.com/coreos/etcd/pkg/ioutil
github.com/rook/rook/vendor/github.com/coreos/etcd/pkg/logutil
github.com/rook/rook/vendor/github.com/coreos/etcd/snap/snappb
github.com/rook/rook/vendor/github.com/xiang90/probing
github.com/rook/rook/vendor/github.com/coreos/etcd/pkg/crc
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/runtime/serializer/json
github.com/rook/rook/vendor/github.com/coreos/etcd/wal/walpb
github.com/rook/rook/vendor/github.com/gogo/protobuf/proto
github.com/rook/rook/vendor/github.com/grpc-ecosystem/grpc-gateway/runtime
github.com/rook/rook/vendor/github.com/coreos/etcd/pkg/cors
github.com/rook/rook/vendor/github.com/coreos/etcd/etcdserver/membership
github.com/rook/rook/vendor/github.com/coreos/etcd/etcdserver/stats
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/runtime/serializer
github.com/rook/rook/vendor/github.com/coreos/etcd/snap
github.com/rook/rook/vendor/github.com/coreos/etcd/rafthttp
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/api
github.com/rook/rook/vendor/github.com/coreos/etcd/etcdserver/api
github.com/rook/rook/vendor/github.com/coreos/etcd/etcdserver/etcdserverpb
github.com/rook/rook/vendor/github.com/coreos/etcd/wal
github.com/rook/rook/vendor/golang.org/x/net/context/ctxhttp
github.com/rook/rook/pkg/util/proc
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/emicklei/go-restful/log
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/emicklei/go-restful
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/blang/semver
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/api/validation/path
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/juju/ratelimit
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/util/clock
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/version
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/util/integer
github.com/rook/rook/vendor/k8s.io/client-go/1.5/tools/clientcmd/api
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/util/flowcontrol
github.com/rook/rook/vendor/k8s.io/client-go/1.5/tools/metrics
github.com/rook/rook/vendor/k8s.io/client-go/1.5/transport
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apimachinery
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apimachinery/registered
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/golang.org/x/oauth2/internal
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/emicklei/go-restful/swagger
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/cloud.google.com/go/internal
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/golang.org/x/oauth2
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/golang.org/x/net/context/ctxhttp
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/cloud.google.com/go/compute/metadata
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/golang.org/x/oauth2/jws
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/golang.org/x/oauth2/jwt
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/coreos/go-oidc/jose
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/coreos/go-oidc/http
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/golang.org/x/oauth2/google
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/coreos/pkg/httputil
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/coreos/go-oidc/oauth2
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/coreos/pkg/health
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/coreos/pkg/timeutil
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/jonboulle/clockwork
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/coreos/go-oidc/key
github.com/rook/rook/vendor/k8s.io/client-go/1.5/vendor/github.com/coreos/go-oidc/oidc
github.com/rook/rook/vendor/github.com/coreos/etcd/client
github.com/rook/rook/vendor/github.com/coreos/etcd/alarm
github.com/rook/rook/vendor/github.com/coreos/etcd/auth
github.com/rook/rook/vendor/github.com/coreos/etcd/compactor
github.com/rook/rook/vendor/github.com/coreos/etcd/lease/leasehttp
github.com/rook/rook/pkg/util
github.com/rook/rook/vendor/github.com/coreos/etcd/discovery
github.com/rook/rook/vendor/github.com/coreos/etcd/etcdserver
github.com/rook/rook/pkg/clusterd/inventory
github.com/rook/rook/vendor/github.com/coreos/etcd/etcdserver/auth
github.com/rook/rook/vendor/github.com/coreos/etcd/etcdserver/api/v2http
github.com/rook/rook/pkg/etcdmgr/bootstrap
github.com/rook/rook/pkg/clusterd
github.com/rook/rook/pkg/cephmgr/mon
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/autoscaling
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/batch
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/certificates
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/apps
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/authorization
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/policy
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/rbac
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/util/cert
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/storage
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apimachinery/announced
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/authentication
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/extensions
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/api/v1
github.com/rook/rook/pkg/operator/k8sutil
github.com/rook/rook/vendor/k8s.io/client-go/1.5/rest
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/apps/v1alpha1
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/authorization/v1beta1
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/autoscaling/v1
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/batch/v1
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/certificates/v1alpha1
github.com/rook/rook/vendor/k8s.io/client-go/1.5/discovery
github.com/rook/rook/vendor/k8s.io/client-go/1.5/kubernetes/typed/authentication/v1beta1
github.com/rook/rook/vendor/k8s.io/client-go/1.5/kubernetes/typed/apps/v1alpha1
github.com/rook/rook/vendor/k8s.io/client-go/1.5/kubernetes/typed/authorization/v1beta1
github.com/rook/rook/vendor/k8s.io/client-go/1.5/kubernetes/typed/autoscaling/v1
github.com/rook/rook/vendor/k8s.io/client-go/1.5/kubernetes/typed/batch/v1
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/policy/v1alpha1
github.com/rook/rook/vendor/k8s.io/client-go/1.5/kubernetes/typed/certificates/v1alpha1
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/extensions/v1beta1
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/rbac/v1alpha1
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/storage/v1beta1
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/api/install
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/apps/install
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/authentication/v1beta1
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/authorization/install
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/autoscaling/install
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/batch/v2alpha1
github.com/rook/rook/vendor/k8s.io/client-go/1.5/kubernetes/typed/storage/v1beta1
github.com/rook/rook/vendor/k8s.io/client-go/1.5/kubernetes/typed/core/v1
github.com/rook/rook/vendor/k8s.io/client-go/1.5/kubernetes/typed/policy/v1alpha1
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/authentication/install
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/certificates/install
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/policy/install
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/storage/install
github.com/rook/rook/vendor/k8s.io/client-go/1.5/kubernetes/typed/rbac/v1alpha1
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/rbac/install
github.com/rook/rook/vendor/k8s.io/client-go/1.5/plugin/pkg/client/auth/gcp
github.com/rook/rook/vendor/k8s.io/client-go/1.5/plugin/pkg/client/auth/oidc
github.com/rook/rook/vendor/k8s.io/client-go/1.5/plugin/pkg/client/auth
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/batch/install
github.com/rook/rook/vendor/k8s.io/client-go/1.5/pkg/apis/extensions/install
github.com/rook/rook/vendor/k8s.io/client-go/1.5/kubernetes/typed/extensions/v1beta1
github.com/rook/rook/vendor/k8s.io/client-go/1.5/kubernetes
github.com/rook/rook/pkg/operator/mon
github.com/rook/rook/pkg/operator/osd
github.com/rook/rook/pkg/operator
github.com/rook/rook/cmd/rook-operator

real	0m59.822s
user	0m0.048s
sys	0m0.032s
```

**Comment by bassam** [comment]
looks like we're not persisting the go/pkg dir when running on linux. it needs to be bind mounted somewhere.

**Comment by bassam** [comment]
Fixed via #350
